chalicelib_smaht/app_utils.py

```diff
@@ -22,11 +22,11 @@ class AppUtils(AppUtils_from_core):
 
     # overwriting parent class
     prefix = FOURSIGHT_PREFIX
-    FAVICON = 'https://cgap-dbmi.hms.harvard.edu/favicon.ico'
+    FAVICON = 'https://data.smaht.org/favicon.ico'
     host = HOST
-    package_name = 'chalicelib_cgap'
+    package_name = 'chalicelib_smaht'
 
-    DEFAULT_ENV = os.environ.get("ENV_NAME", "foursight-cgap-env-uninitialized")
+    DEFAULT_ENV = os.environ.get("ENV_NAME", "foursight-smaht-env-uninitialized")
```

Comment:
Small matter, but should this just reference `APP_PACKAGE_NAME`? Just thinking aloud here. Maybe that's not the right thing. But what I was thinking was something like this. I feel like there shouldn't be as many wired-in places where the name is mentioned as there are. I won't mark them all but am using this as an example.
```
DEFAULT_ENV = os.environ.get("ENV_NAME", f"{APP_PACKAGE_NAME}-env-uninitialized")
```